src/_premake_init.lua

```diff
@@ -467,6 +467,18 @@
 	}
 
 	api.register {
+		name = "inlining",
+		scope = "config",
+		kind = "string",
+		allowed = {
+			"Default",
+			"Disabled",
+			"Explicit",
+			"Implicit"
```

Comment:
"Explicit" -> "WheneverISaySo"
"Implicit" -> "WhateverTheCompilerThinksIsRight"

But I'm open to suggestions...